google/identity/accesscontextmanager/v1/access_level.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2022 Google LLC
```

Comment:
Q: just checking this is the correct date.

Reply:
I believe it is correct because the file was copied directly from here:
https://github.com/googleapis/googleapis/blob/b36b446bf4ec1e9980bb82bb86a8471513bc5054/google/identity/accesscontextmanager/v1/access_level.proto#L1